.github/workflows/tembo_ai.yaml

```diff
@@ -50,17 +50,15 @@ jobs:
         with:
           toolchain: stable
       - name: install dependencies
-        run: cargo install sqlx-cli --version 0.7.4
```

Comment:
updating sqlx-cli version, and running the tests using the sqlx cache.  Running the tests using sqlx offline, and running migrations executed from webserver code.  Then putting `make check` at the end after the tests, so that we don't have to explicitly run the migrations.